aten/src/ATen/native/quantized/cpu/fbgemm_utils.h

```diff
@@ -15,17 +20,19 @@
 // of the A rows. The column offsets are needed for the asymmetric quantization
 // (affine quantization) of input matrix.
 // Note that in JIT mode we can think of a way to fuse col_offsets with bias.
+template <typename T = fbgemm::PackBMatrix<int8_t>>
 struct FBGEMM_API PackedLinearWeight {
```

Comment:
seems like MKLDNN uses only a few fields here - bias and w. I suggest creating a separate structure instead of templating this one (and it'd be much more comprehensible for code organization and fbgemm_utils wouldn't include mkldnn)

Reply:
I had separated the mkldnn-related code to qmkldnn_utils.h.